charts/jenkins/values.yaml

```
@@ -241,7 +241,7 @@ controller:
     - kubernetes:3600.v144b_cd192ca_a_
     - workflow-aggregator:581.v0c46fa_697ffd
     - git:4.11.3
```

```suggestion
    - kubernetes:3706.vdfb_d599579f3
    - workflow-aggregator:590.v6a_d052e5a_a_b_5
    - git:4.11.5
```
Updating all installed plugins